egs/wsj/s5/steps/segmentation/get_targets_for_out_of_segments.sh

```diff
@@ -0,0 +1,81 @@
+#! /bin/bash
+
+# Copyright 2017  Vimal Manohar
+# Apache 2.0
+
+nj=4
+cmd=run.pl
+default_targets=   # vector of default targets in text format
+frame_subsampling_factor=1
+
+set -o pipefail -u
+
+[ -f ./path.sh ] && . ./path.sh
+. utils/parse_options.sh
+
+if [ $# -ne 3 ]; then
+  cat <<EOF
+  Usage: steps/segmentation/get_targets_for_out_of_segments.sh <data-dir> <whole-data-dir> <targets-dir>
```

Comment:
It's better to have it separate so that the user can weight the real supervision from alignment and the assumed default targets in the out of segment regions differently. I can remove the default-targets option from convert-targets-to-whole if needed.